LICENSE.md

```diff
@@ -8,7 +8,7 @@ Version 2.0.2
 
 **Game**: Minecraft, the Minecraft server, and all associated software or derivatives, as defined by Mojang.
 
-**Mod**: Software which modifies the Game, in any form (source, compiled binary, etc).
+**The Mod**: Any copyrightable work licensed under this license, in any form (source, compiled binary, etc).
```

Comment:
I meant that in the license text sometimes "the mod", sometimes "this mod" is used. Maybe change everything to "this mod".